src/main/java/com/metamx/collections/bitmap/RoaringBitmapFactory.java

```diff
@@ -58,9 +61,7 @@ public MutableBitmap makeEmptyMutableBitmap()
   @Override
   public ImmutableBitmap makeEmptyImmutableBitmap()
   {
-    return new WrappedImmutableRoaringBitmap(
-        EMPTY_IMMUTABLE_BITMAP
-    );
+    return WRAPPED_IMMUTABLE_ROARING_BITMAP;
```

Comment:
`org.roaringbitmap.buffer.ImmutableRoaringBitmap#getIntIterator` has a construction of a shared iteration in `ImmutableRoaringIntIterator`  It is not immediately obvious that a single static immutable roaring can be used.